docs/src/pages/components/links/links.md

```diff
@@ -16,24 +16,24 @@ You can leverage its properties.
 However, the Link has different default properties than the Typography:
```

Comment:
```suggestion
However, the Link component has different default properties than the Typography component:
```
? A bit longer but I think it flows nicer.

Reply:
Yep.